tools/monitoring/README.md

```diff
@@ -28,14 +28,21 @@ monitoring_service file to make any updates.
 
 ### Update Image
 
-1. `images/monitoring/Makefile` Commands to build and deploy the monitoring
-   images.
+1. Build and deploy the monitoring images.
+   ```bash
+   cd images/monitoring/
+   make build
+   make push_versioned
+   ```
+
+1. Update `tools/monitoring/gke_deployment/monitoring_service.yaml` to use the image tag created in previous step
 
 1. Update to use the latest image on GKE
 
    ```bash
    kubectl rollout restart deployment.apps/monitoring-app
    ```
+   Note that if the "restart" sub-command is not found, update your kubectl installation.
```

Comment:
Since we don't need to use `kubectl rollout` anymore, a simple check on the status of the components should be enough i.e. `kubectl get all` or `kubectl get pods`.